tests/find_cmd_tests.rs

```diff
@@ -518,3 +518,38 @@ fn expression_empty_parentheses() {
         ))
         .stdout(predicate::str::is_empty());
 }
+
+#[test]
+fn find_age_range() {
+    let args = ["-amin", "-cmin", "-mmin"];
+    let times = ["-60", "-120", "-240", "+60", "+120", "+240"];
+    let time_strings = [
+        "\"-60\"", "\"-120\"", "\"-240\"", "\"-60\"", "\"-120\"", "\"-240\"",
+    ];
+
+    for arg in args.iter() {
+        for time in times.iter() {
+            Command::cargo_bin("find")
+                .expect("the time should match")
+                .args(["test_data/simple", arg, time])
+                .assert()
+                .success()
+                .code(0);
+        }
+    }
+
+    for arg in args.iter() {
+        for time_string in time_strings.iter() {
+            Command::cargo_bin("find")
+                .expect("the except time should not match")
```

Comment:
I'm not sure about this one:
```suggestion
                .expect("the time should not match")
```